### DISCUSSION TITLE
LFS files in archive, billing?

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#22426] [New to GitHub]
<p>In the settings, I can choose whether to</p>
<pre><code class="lang-auto">[ ] Include Git LFS objects in archives
Git LFS usage in archives is billed at the same rate as usage with the client.
</code></pre>
<p>I’m not sure what that last sentence means. Is that just for storage, or does every archive download count towards the total the bandwidth, too?</p>

**Comment by tachuris** [comment, marked as answer]
<blockquote>
<p>does every archive download count towards the total the bandwidth, too?</p>
</blockquote>
<p>I believe that’s the case:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/billing/managing-billing-for-git-large-file-storage/about-billing-for-git-large-file-storage">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/billing/managing-billing-for-git-large-file-storage/about-billing-for-git-large-file-storage" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/114728/181070852-32c387e5-ad5e-4828-9632-fba49e0febc0.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/billing/managing-billing-for-git-large-file-storage/about-billing-for-git-large-file-storage" target="_blank" rel="noopener nofollow ugc">About billing for Git Large File Storage - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
